### PR TITLE
1-stream-1-table (Destinations v2) `destination-bigquery` beta testing publish PR

### PR DESCRIPTION
**Pull request by evantahler**
## ⚠️ This PR should never be merged ⚠️

This PR enables the secret config option to allow folks to beta-test `destination-bigquery` + 1s1t (Destinations V2).  We will be using this PR to publish dev/beta images of the connector.

### Steps for updating the Beta Image

1. Create a development branch based off of `master` and make your changes, get your PR reviewed, (generally follow the normal process for all changes)
2. After your PR from step 1 is merged into `master`, merge the newly updated `master` into `destinations-v2-beta`. An easy way to do this is clicking the `Update branch` button at the bottom of this PR.
3. Run [this GitHub Workflow](https://github.com/airbytehq/airbyte/actions/workflows/publish_connectors.yml) with the following options
    - branch: `destinations-v2-beta`
    - options: `--name=destination-bigquery`
    - leave the `--pre-release` flag set and use whatever the `runs-on` option is set to
4. Once the Github Workflow completes, Update this PR's Beta Releases Table in the description
5. Verify the image is available in [DockerHub](https://hub.docker.com/repository/docker/airbyte/destination-bigquery/general)
6. Update the feature flag to use the correct image tag variant go to the [Variations](https://app.launchdarkly.com/default/production/features/connectors.versionOverrides/variations) page, search for `Bigquery 2.0.0-destinationsv2-beta` which is currently "Variation 25" and set its "value" to the image tag, e.g. `1.5.7-dev.7e4a9dd278`
7. Also make sure OSS beta testers are informed of the new image

Feature flag things to add the new pre-release docker image to the workspaces you want to test.  You will need to change the tag on the varaint each release.  [Internal Docs](https://internal-docs.airbyte.io/How-Tos/Connector-Version-Overrides/) on how to update the feature flag

## Beta Releases

| Date | Who | GH Action | Version |
|--------|--------|--------|--------|
| 2023-06-29 | @evantahler | [Link](https://github.com/airbytehq/airbyte/actions/runs/5415948291) | [airbyte/destination-bigquery:1.5.0-dev.7a773a7766](https://hub.docker.com/r/airbyte/destination-bigquery/tags) |
| 2023-07-05 | @edgao  | [Link](https://github.com/airbytehq/airbyte/actions/runs/5469974586) | [airbyte/destination-bigquery:1.5.2-dev.27fd242fda](https://hub.docker.com/r/airbyte/destination-bigquery/tags) |
| 2023-07-10 | @edgao  | [Link](https://github.com/airbytehq/airbyte/actions/runs/5513488745/jobs/10051694654) | [airbyte/destination-bigquery:1.5.2-dev.c27684e40d](https://hub.docker.com/layers/airbyte/destination-bigquery/1.5.2-dev.c27684e40d/images/sha256-735a1f581fedaf26c5314b9f3da5a8a9394ea323db145923a6d35712ee9f1c74?context=explore) |
| 2023-07-11 | @edgao  | [Link](https://github.com/airbytehq/airbyte/actions/runs/5524075381/jobs/10075868245) | [airbyte/destination-bigquery:1.5.2-dev.8a8cb3539f](https://hub.docker.com/layers/airbyte/destination-bigquery/1.5.2-dev.8a8cb3539f/images/sha256-e0b7c8e810f33419bcd9fe62ee10c0f7425537569542db673ac3291e7f58cf60?context=explore) |
| 2023-07-13 | @evantahler  | (manual) | [airbyte/destination-bigquery:1.5.2-dev.8cacb66bdb](https://hub.docker.com/layers/airbyte/destination-bigquery/1.5.2-dev.8cacb66bdb/images/sha256-9be37dfae6203045632b234ca4a6c24c9893ea41fc7925161cd8213c493ea0ac?context=explore) |
| 2023-07-18 | @edgao  | (manual) | [airbyte/destination-bigquery:1.5.3-dev.d5d7e757f8](https://hub.docker.com/layers/airbyte/destination-bigquery/1.5.3-dev.d5d7e757f8/images/sha256-58c7a29afa5e077b294dd768b58f6e2fddbbd87a53cbe75114a125ff0ff6322a?context=repo) |
| 2023-07-20 | @evantahler | [link](https://github.com/airbytehq/airbyte/actions/runs/5613553429/job/15209832233) | [airbyte/destination-bigquery:1.5.4-dev.da21b754b1](https://hub.docker.com/r/airbyte/destination-bigquery/tags) | 
| 2023-07-21 | @edgao | [link](https://github.com/airbytehq/airbyte/actions/runs/5627272502/job/15249830992) | [airbyte/destination-bigquery:1.5.6-dev.f80318f754](https://hub.docker.com/layers/airbyte/destination-bigquery/1.5.6-dev.f80318f754/images/sha256-9571e0436ab070fdeeeaf1a1a37ae811933b602fcfd41642f7f361749ad11f7e?context=repo) | 
| 2023-07-24 | @jbfbell | [link](https://github.com/airbytehq/airbyte/actions/runs/5650137605) | [airbyte/destination-bigquery/1.5.7-dev.7e4a9dd278](https://hub.docker.com/layers/airbyte/destination-bigquery/1.5.7-dev.7e4a9dd278/images/sha256-64244bf3b0e971ff2074017604c6f07bd5e7112d6b1c0934b88962a1537d5f90?context=repo) |
| 2023-07-27 | @edgao | [link](https://github.com/airbytehq/airbyte/actions/runs/5683766137/job/15405017305) | [airbyte/destination-bigquery/1.6.0-dev.281a111a65](https://hub.docker.com/layers/airbyte/destination-bigquery/1.6.0-dev.281a111a65/images/sha256-82c2a1711f7355edc073047e8dfa862e2a34988c21902e0be9744a1e8730d002?context=explore) |

### CODE REVIEW

**Comment by edgao** [comment]
/legacy-publish connector=connectors/destination-bigquery run-tests=false

**Comment by edgao** [comment]
/legacy-publish connector=connectors/destination-bigquery run-tests=false --pre-release

**Comment by edgao** [comment]
/legacy-publish connector=connectors/destination-bigquery run-tests=false --pre-release

**Comment by edgao** [comment]
/legacy-publish connector=connectors/destination-bigquery run-tests=false --pre-release=true

**Comment by edgao** [comment]
/legacy-publish connector=connectors/destination-bigquery run-tests=false pre-release=true

File: airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json [hunk 1]
```diff
@@ -11,6 +11,11 @@
     "required": ["project_id", "dataset_location", "dataset_id"],
     "additionalProperties": true,
     "properties": {
+      "use_1s1t_format": {
+        "type": "boolean",
+        "description": "(Beta) Use <a href=\"https://github.com/airbytehq/airbyte/issues/26028\" target=\"_blank\">Destinations V2</a>. Contact Airbyte Support to participate in the beta program.",
+        "title": "Use Destinations V2 (Advanced)"
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json:17]:
```suggestion
        "title": "Use Destinations V2 (Early Access)"
```

File: airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json [hunk 2]
```diff
@@ -11,6 +11,11 @@
     "required": ["host", "role", "warehouse", "database", "schema", "username"],
     "additionalProperties": true,
     "properties": {
+      "use_1s1t_format": {
+        "type": "boolean",
+        "description": "(Beta) Use <a href=\"https://github.com/airbytehq/airbyte/issues/26028\" target=\"_blank\">Destinations V2</a>. Contact Airbyte Support to participate in the beta program.",
+        "title": "Use Destinations V2 (Advanced)"
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json:17]:
```suggestion
        "title": "Use Destinations V2 (Early Access)"
```

**Comment by evantahler** [comment]
This PR is closed because destination-bigquery + Destination V2 is now available behind a config option on master.

**Pull request closed without merging**